### DISCUSSION TITLE
Mutagenesis: replace an amino acid residue with another one

### DISCUSSION THREAD

**Discussion by slowkow** [molstar/molstar#676] [Ideas]
Mutagenesis enables users to select an amino acid residue in the current protein and replace it with another amino acid residue. This is an important feature for genetics researchers who want to visualize the 3-dimensional structure of a mutation.

PyMOL provides a helpful [wizard for mutagenesis](https://pymolwiki.org/index.php/Mutagenesis) that enables users to replace one amino acid residue with another amino acid.

This video demonstrates the functionality: https://www.youtube.com/watch?v=M-VCBz83nfs

Many users would be happy to see a similar feature supported by molstar!

**Comment by spencel** [comment]
It seems like the best workaround for now is to use PyMOL API and then load that modified protein structure data into Molstar?
